src/server/api/file/index.ts

```diff
@@ -38,18 +39,40 @@ export const FileApi = {
     express.get(ApiEndPoint.File.biomassStock({}), AuthMiddleware.requireEditTableData, getBiomassStockFile)
 
     // Files
+    express.post(
+      ApiEndPoint.File.many(),
+      multer({ fileFilter }).array('file'),
+      AuthMiddleware.requireEditAssessmentFile,
+      createManyFiles
+    )
```

Comment:
change: this is the only change, all are deprecation reminders (will clean up all deprecations later prs)